package-lock.json

```diff
@@ -3931,6 +3931,12 @@
       "resolved": "https://registry.npmjs.org/arr-union/-/arr-union-3.1.0.tgz",
```

Comment:
Were all the changes in this file intended?

Reply:
yeah, we just deleted it and then let it get created

Reply:
No, but this is likely the result of failures to commit this file from earlier library additions. No NPM updates were run, only installs.